app/ui.js

```diff
@@ -98,6 +98,10 @@ var UI;
         lastKeyboardinput: null,
         defaultKeyboardinputLen: 100,
 
+        reconnect: false,
+        reconnect_delay: 5000,
```

Comment:
You should be using our settings API instead of global variables for "reconnect" and "reconnect_delay"